pkg/config/config.go

```diff
@@ -0,0 +1,87 @@
+package config
+
+import "context"
+
+// Log defines the available logging configuration.
+type Log struct {
+	Level  string
+	Pretty bool
+	Color  bool
+	File   string
+}
+
+// Debug defines the available debug configuration.
+type Debug struct {
+	Addr   string
+	Token  string
+	Pprof  bool
+	Zpages bool
+}
+
+// HTTP defines the available http configuration.
+type HTTP struct {
+	Addr      string
+	Namespace string
+	Root      string
+	CacheTTL  int
+}
+
+// GRPC defines the available grpc configuration.
+type GRPC struct {
+	Addr      string
+	Namespace string
+}
+
+// Server configures a server.
+type Server struct {
+	Version string
+	Name    string
+}
+
+// Tracing defines the available tracing configuration.
+type Tracing struct {
+	Enabled   bool
+	Type      string
+	Endpoint  string
+	Collector string
+	Service   string
+}
+
+// Asset defines the available asset configuration.
+type Asset struct {
+	Path string
+}
+
+// TokenManager is the config for using the reva token manager
+type TokenManager struct {
+	JWTSecret string
+}
+
+type WopiServer struct {
+	WopiServerHost     string
```

Comment:
since the struct is named `WopiServer`, how about naming the attributes `Host`, `Secret` and `Insecure`? :)